dataflow/src/main/java/org/checkerframework/dataflow/cfg/builder/CFGTranslationPhaseOne.java

```diff
@@ -1269,13 +1269,15 @@ protected Node methodInvocationConvert(Node node, TypeMirror formalType) {
      * a list of {@link Node}s representing the arguments converted for a call of the method. This
      * method applies to both method invocations and constructor calls.
      *
+     * @param enclosingType a TypeMirror of the enclosing expression if appropriate
```

Comment:
Would `enclosingExprType` be a clearer name? Otherwise, a quick read of the name could make one think it's the type that contains this method.

Reply:
Explain when it is "appropriate" and when it is null.
It might be worth expanding on this in the documentation above.

Reply:
Thanks. Done in my local.

Reply:
Sorry about that, I pushed it just now.